### PR TITLE
Migrate Button example to Args

### PR DESCRIPTION
**Pull request by baiirun**
## Description
npm 7 now automatically installs peer dependencies if you don't pass the `--legacy-peer-deps` flag. Some of the peer dependencies installed in our React examples don't correctly match.

### CODE REVIEW

File: react/storybook-v6/src/components/Button.jsx [hunk 1]
```diff
@@ -4,22 +4,29 @@ import { ChevronRightIcon } from './icons';
...
+};
+
 /**
  * Buttons indicate actions on the page that will occur when a user touches them. This button is responsive to screen sizes.
  * We support a very minimal level of variation for the primary button. Custom styling or class names should not be applied to the primary button.
  * It requires text in all cases, but allows for icons as an optional addition.
  * The primary button can also exist as an element within larger molecules and organisms.
  * */
 const Button = ({ onClick, icon, disabled, children }) => {
+  const IconComponent = iconOptions[icon];
```
Comment (ByronGuina) [inline react/storybook-v6/src/components/Button.jsx:19]: I updated some of the examples in React w/ Storybook 6 to use Args.

File: react/storybook-v6/src/components/Button.stories.js [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
+    {children}
   </Button>
 );
 
+simpleButton.args = {
+  disabled: false,
+  icon: iconOptions.none,
+  children: 'Button'
+};
+
```
Comment (ByronGuina) [inline react/storybook-v6/src/components/Button.stories.js:32]: I updated some of the examples in React w/ Storybook 6 to use Args.

**Comment by baiirun** [comment]
> Is there any plan to merge this? Or generally update the plugin to Storybook 6/7 and React 18 ? @baiirun

Unfortunately I no longer work at DSM so I'm not sure the state of this repo. I'll close for now and the team can re-open from this branch if necessary.

**Comment by baiirun** [comment]
This PR has been open for a while, and since I don't work at InVision anymore I'm not sure the current state of the roadmap. Going to close for now.

**Pull request closed without merging**